Firestore/Protos/protos/firestore/local/mutation.proto

```diff
@@ -55,4 +55,16 @@ message WriteBatch {
 
   // The local time at which the write batch was initiated.
   google.protobuf.Timestamp local_write_time = 3;
+
+  // A list of "writes" that represent a partial base state from when this
+  // write batch was initially created. During local application of the write
```

Comment:
"local application" can have a specific meaning within our application but it's ambiguous without full context (it could also conceivably mean computing a dirty document after a write acknowledgement). I'd suggest a phrase like "when computing the local view", which avoids the ambiguity.